tools/rosbag/include/rosbag/player.h

```diff
@@ -177,17 +178,24 @@ class ROSBAG_DECL Player
 
     void printTime();
 
+    bool pauseCallback(std_srvs::SetBool::Request &req, std_srvs::SetBool::Response &res);
 
 private:
 
     PlayerOptions options_;
 
     ros::NodeHandle node_handle_;
 
+    ros::ServiceServer pause_service_;
```

Comment:
I guess this changes the ABI, but I don't think anyone is relying on that @dirk-thomas ?